pandas/core/sparse/array.py

```diff
@@ -290,7 +290,9 @@ def __reduce__(self):
         """Necessary for making this object picklable"""
         object_state = list(np.ndarray.__reduce__(self))
         subclass_state = self.fill_value, self.sp_index
-        object_state[2] = (object_state[2], subclass_state)
+        object_state[2] = list(object_state[2])
```

Comment:
Just curious: why the changes here? Did we have tests that failed without it?

Reply:
Yes, `test_pickle` was failing.`shape` is used to pickle objects and the overridden `shape` does not return the correct value. Therefore, I had to set the `shape` manually.